tests/integration/test_app_create.py

```diff
@@ -227,6 +229,50 @@ def test_defaults_to_adding_the_log_submitter_to_the_application(self):
         # Then
         assert isinstance(application.eq["submitter"], LogSubmitter)
 
+    def test_eq_publisher_backend_not_set(self):
+        # Given
+        self._setting_overrides["EQ_PUBLISHER_BACKEND"] = ""
+
+        # When
+        with self.assertRaises(Exception) as ex:
+            create_app(self._setting_overrides)
+
+        # Then
+        assert "Unknown EQ_PUBLISHER_BACKEND" in str(ex.exception)
+
+    def test_adds_pub_sub_to_the_application(self):
+        # Given
+        self._setting_overrides["EQ_PUBLISHER_BACKEND"] = "pubsub"
+        self._setting_overrides["EQ_PUB_SUB_TOPIC_ID"] = "123"
+
+        # When
+        with patch(
+            "app.publisher.publisher.google.auth._default._get_explicit_environ_credentials",
+            return_value=(Mock(), "test-project-id"),
+        ):
+            application = create_app(self._setting_overrides)
+
+        # Then
+        assert isinstance(application.eq["publisher"], PubSub)
+
+    def test_pub_sub_topic_id_not_set_raises_exception(self):
+        # Given
+        self._setting_overrides["EQ_PUBLISHER_BACKEND"] = "pubsub"
+
+        # WHEN
```

Comment:
This must be an important WHEN

Reply:
The one I copied must have also been important 😄 . Have updated.